src/test/java/duke/tasks/EventTest.java

```diff
@@ -0,0 +1,15 @@
+package duke.tasks;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.time.LocalDate;
+import duke.tasks.Event;
+
+public class EventTest {
+    @Test
+    public void testToString(){
+        assertEquals("[E][ ] finish homework (from: 2pm to: 4pm)", new Event("finish homework", "from 2pm to 4pm").toString());
+    }
+
+}
```

Comment:
I think you can include a new line here. I don't think its compulsory but github warns you about it. I found a reason online in this link [here](https://www.quora.com/Why-is-it-a-common-practice-to-put-a-new-line-at-the-end-of-a-source-code-file) if you're interested.